src/lib/core/port.h

```diff
@@ -33,6 +33,8 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "box/sql/sqlInt.h"
```

Comment:
Please drop `#include "box/sql/sqlInt.h"` and add `struct Mem;` just after `struct port;` a little further down.

Reply:
Added.